app/views/events/participants.html.erb

```diff
@@ -32,6 +23,13 @@
       <%= participant.email %>
     </td>
 	<td>
+	  <% if participant.older_than_18_at_start_date_of_event?(@event) %>
+	    <p class="text-muted"> <%= t '.unnecessary' %> </p>
+	  <% elsif participant.agreement_letter_for_event?(@event) %>
+	    <p class="text-success" title="<%= participant.agreement_letter_for_event(@event).updated_at %>"> <%= t '.available' %> </p>
```

Comment:
Requirement in User Story ("While holding the mouse on the symbol, you can see the date and the time of the upload")